tarantoolctl

```diff
@@ -0,0 +1,1353 @@
+#!/usr/bin/env tarantool
```

Comment:
I guess that only start/stop/status is needed for test-run, not all the 1353 lines of the code. It should be easy to strip unneeded parts.

Reply:
My experience in administration tells me that when we migrate something (`state1 -> state2`), it is necessary to save states and not change something. I have created a ticket and we will be able to return to this issue a little later.

Reply:
Agree with @ochaplashkin.

Reply:
OK, no problem.